NGitLab.Mock/Config/GitLabHelpers.cs

```diff
@@ -1203,9 +1250,10 @@ private static void CreateGroup(GitLabServer server, GitLabGroup group)
 
     private static void CreateProject(GitLabServer server, GitLabProject project)
     {
-        var prj = new Project(project.Name ?? Guid.NewGuid().ToString("D"))
+        var prj = new Project(project.Name ?? project.Path ?? Guid.NewGuid().ToString("D"))
         {
             Id = project.Id,
+            Path = project.Path,
```

Comment:
Good observation. I'm glad to know you are paying attention! 😉 You are correct: `config.WithProject("Test", configure: p => p.@Namespace = "Foo")` was replacing the default path with null. I have corrected this. However, it ultimately didn't matter because `Mock.ProjectCollection.Add()` auto-generates the path from the name, and vice-versa, just like GitLab does.